walnuts_cpp/nuts.hpp

```diff
@@ -17,19 +18,20 @@ using Integer = std::int32_t;
 template <typename S, class Generator>
 class Random {
  public:
-  Random(Generator& rng): rng_(rng), unif_(0.0, 1.0), binary_(0.5), normal_(0.0, 1.0) { }
+  explicit Random(Generator& rng): rng_(rng), unif_(0.0, 1.0), binary_(0.5), normal_(0.0, 1.0) { }
 
-  S uniform_real_01() {
+  inline S uniform_real_01() {
     return unif_(rng_);
   }
 
-  bool uniform_binary() {
+  inline bool uniform_binary() {
     return binary_(rng_);
   }
 
-  Vec<S> standard_normal(Integer n) {
+  inline Vec<S> standard_normal(Integer n) noexcept {
```

Comment:
I don’t think the noexcept is correct here, nor do I think it adds much

Reply:
Oops.  I meant to remove that when I realized that the embedded normals weren't `noexcept`.  It's mainly for clarity---I know it's not going to be a big deal on efficiency.

Reply:
Thanks or catching.  I removed the `noexcept`.  I realized it was a mistake because the RNGs aren't marked `noexcept`---just missed this one because of the long line.